resources/views/platform/components/game/current-progress/progress-bar.blade.php

```diff
@@ -16,11 +16,11 @@
     $hardcoreProgressWidth = ($numEarnedHardcoreAchievements / $totalAchievementsCount) * 100;
     $softcoreProgressWidth = ($numEarnedSoftcoreAchievements / $totalAchievementsCount) * 100;
 
-    $hardcoreCompletionPercentage = sprintf("%01.0f", floor($hardcoreProgressWidth));
-    $softcoreCompletionPercentage = sprintf("%01.0f", floor($softcoreProgressWidth + $hardcoreCompletionPercentage));
+    $hardcoreCompletionPercentage = sprintf("%01.0f", $hardcoreProgressWidth);
+    $softcoreCompletionPercentage = sprintf("%01.0f", $softcoreProgressWidth + $hardcoreCompletionPercentage);
 }
 
-$completionPercentage = sprintf("%01.0f", floor($hardcoreProgressWidth + $softcoreProgressWidth));
+$completionPercentage = sprintf("%01.0f", round($hardcoreProgressWidth + $softcoreProgressWidth));
```

Comment:
I might be misunderstanding but isn't rounding to floor preferable to not ever display 100% when it's not?

Reply:
Good call. Yes, I think you are right.

In latest, I've reverted the change to the game page progress component and updated the game list item progress bar component. Now the game list item progress should align to the game page instead of the other way around.